### ISSUE TITLE
GanttChart: ColorContrast for the timeline tick-text is lower than 4.5:1

### ISSUE THREAD

**Issue by sulabh-msft** [mermaid-js/mermaid#995]
**Describe the bug**
GanttChart: ColorContrast for the timeline tick-text is lower than 4.5:1

**To Reproduce**
1. Go to  [mermaid live editor with gantt chart default theme](https://mermaidjs.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ2FudHRcbiAgICB0aXRsZSBBIEdhbnR0IERpYWdyYW1cbiAgICBkYXRlRm9ybWF0ICBZWVlZLU1NLUREXG4gICAgc2VjdGlvbiBTZWN0aW9uXG4gICAgQSB0YXNrICAgICAgICAgICA6YTEsIDIwMTQtMDEtMDEsIDMwZFxuICAgIEFub3RoZXIgdGFzayAgICAgOmFmdGVyIGExICAsIDIwZFxuICAgIHNlY3Rpb24gQW5vdGhlclxuICAgIFRhc2sgaW4gc2VjICAgICAgOjIwMTQtMDEtMTIgICwgMTJkXG4gICAgYW5vdGhlciB0YXNrICAgICAgOiAyNGQiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9fQ)
2. Use any color contrast analyzer ib tge tick text ( time-tick at the bottom) ( I used Accessibility Insights for windows
![image](https://user-images.githubusercontent.com/13728069/66827867-49ca9e00-ef6d-11e9-9565-cc9164eb5d5c.png)

**Expected:​**
The colour contrast of the text present for the image should be equal or greater than 4.5:1

**​Actual:​**
The colour contrast of the text for the image is below 4.5:1

**User Impact:​**
Low vision users will not see the text clearly.

**Screenshots**
![image](https://user-images.githubusercontent.com/13728069/66827245-03c10a80-ef6c-11e9-8961-5f89d2402063.png)

**Desktop (please complete the following information):**
 - OS: [ Windows etc.]
 - Browser [ chrome, Edge etc]

**Additional context**
Same issue is in  dark theme as well
[Gantt chart dark theme color contrast issue](https://mermaidjs.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ2FudHRcbiAgICB0aXRsZSBBIEdhbnR0IERpYWdyYW1cbiAgICBkYXRlRm9ybWF0ICBZWVlZLU1NLUREXG4gICAgc2VjdGlvbiBTZWN0aW9uXG4gICAgQSB0YXNrICAgICAgICAgICA6YTEsIDIwMTQtMDEtMDEsIDMwZFxuICAgIEFub3RoZXIgdGFzayAgICAgOmFmdGVyIGExICAsIDIwZFxuICAgIHNlY3Rpb24gQW5vdGhlclxuICAgIFRhc2sgaW4gc2VjICAgICAgOjIwMTQtMDEtMTIgICwgMTJkXG4gICAgYW5vdGhlciB0YXNrICAgICAgOiAyNGQiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGFyayJ9fQ)

**Comment by eel87** [comment]
Hi @Dunning-Kruger is this issue related to mermaid-live-editor or mermaid?  I'm a new developer, interested in helping out with accessibility features but I'm a little confused as to where these changes would be made.  Any advice to get started if this issue is still open?

**Comment by Dunning-Kruger** [comment]
@eel87 This issue is with the default theme (by mermaid) of the Gantt chart. I believe all theme and styling info is here: https://github.com/mermaid-js/mermaid/tree/develop/src/themes

**Comment by eel87** [comment]
@Dunning-Kruger Thank you! I'd like to take stab at this if that's okay.

**Comment by Dunning-Kruger** [comment]
@eel87 We're always welcoming new contributors. Feel free to read read our [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/master/CONTRIBUTING.md) and [join us in slack](https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE)!
